wiki/beginners.md

```diff
@@ -0,0 +1,450 @@
+---
+title: Beginners
+parent: Wiki
+---
+
+This page was moved from the old wiki and is in the process of being revised.
+{: .fs-5 .ls-10 .code-example }
+
+## Basic information
+
+When in doubt, consult the [API
+docs](https://remotestorage.github.io/remotestorage.js/files/remotestorage-js.html),
+or ask on IRC (\#remotestorage on Freenode), or on [the
+forums](https://community.remotestorage.io/).
+
+...
+
+## Get started
+
+### Choose a build
+
+There are a number of different builds available.
+
+#### Standard
+
+Files `remotestorage.js` and `remotestorage.min.js` (minified version).
+
+Contains all components of remotestorage.js for running in a browser.
+
+#### Without local cache
+
+Files `remotestorage-nocache.js` and `remotestorage-nocache.min.js`.
+
+Versions without any caching features included. Use this if you want
+your app to write directly to the remote server without caching any data
+in the browser’s storage (localStorage or IndexedDB).
+
+*Hint: Even if you "disable" or not enable caching for paths in the
+normal build, it will still cache outgoing changes. The only way to make
+sure something landed on the server immediately is to use this no-cache
+build.*
+
+#### AMD
+
+Files `remotestorage.amd.js` (with cache) and
+`remotestorage-nocache.amd.js` (without cache, see
+[below](#Without_local_cache "wikilink")).
+
+remoteStorage.js wrapped for use with
+[AMD](https://en.wikipedia.org/wiki/Asynchronous_module_definition)
+loaders such as [RequireJS](http://requirejs.org/).
+
+### Get remoteStorage.js
+
+#### Bower
+
+`bower install -S remotestorage`
```

Comment:
Suggesting to use a whole line code block, either with prepending four spaces before the line, and leaving out the tick marks, or by wrapping the line into three-tick-marks-blocks.